frontend/src/employee-frontend/components/unit/unit-reservations/StaffAttendanceTable.tsx

```diff
@@ -313,6 +313,9 @@ const StaffAttendanceModal = React.memo(function StaffAttendanceModal({
     <StaffAttendanceDetailsModal
       isExternal={true}
       onSave={onSaveExternal}
+      previousStaffOccupancyEffect={
+        detailsModalConfig.attendances[0]?.occupancyCoefficient > 0 ?? true
```

Comment:
I think this has a bug because `undefined > 0` is `false` and therefore `?? true` is never executed

Reply:
otherwise LGTM

Reply:
Default value assessment fixed. Idea is to avoid using potentially variable coefficient values (e.g. 7) in UI code.